custom_components/spotcast/manifest.json

```diff
@@ -12,6 +12,9 @@
     "dependencies": [
         "spotify"
     ],
+    "requirements": [
```

Comment:
move after `issue_tracker`. Manifest Keys must be alphabetical (I have no idea why that's an issue Its a requirement to pass hassfest validation)